genai/internal/samples/docs-snippets_test.go

```diff
@@ -40,13 +40,23 @@ var testDataDir = filepath.Join(testhelpers.ModuleRootDir(), "genai", "testdata"
 
 func ExampleGenerativeModel_GenerateContent_textOnly() {
 	ctx := context.Background()
-	client, err := genai.NewClient(ctx, option.WithAPIKey(os.Getenv("GEMINI_API_KEY")))
+	client, err := genai.NewClient(ctx, option.WithAPIKey(os.Getenv("GOOGLE_API_KEY")))
 	if err != nil {
 		log.Fatal(err)
 	}
 	defer client.Close()
 
 	// [START text_gen_text_only_prompt]
+	import "github.com/google/generative-ai-go/genai"
```

Comment:
Import lines must be at the top of the file, just below the package declaration.